server/statistics/region_stat_informer.go

```diff
@@ -0,0 +1,11 @@
+package statistics
+
+import "github.com/pingcap/pd/server/core"
+
+// RegionStatInformer provides access to a shared informer of statistics.
+type RegionStatInformer interface {
```

Comment:
Is this necessary to separate these two interfaces into different files?

Reply:
It's not necessary, but there is no reason to put them together.